content/release-10-15-0/whatsnew-10-15-0.md

```diff
@@ -41,3 +41,21 @@ The Sigfox and Actility agents now support the configuration of multiple connect
 ![All devices](/images/release-notes/multiple_lns_connection.png)
 
 For details, refer to [Sigfox](https://cumulocity.com/guides/protocol-integration/sigfox) or [Actility LoRa](https://cumulocity.com/guides/protocol-integration/lora-actility) in the *Protocol integration guide*.
+
+### Device Management
+
+#### Revised LWM2M device registration
+
+The LWM2M agent now provides revised single- and device bulk registration wizards. These dialogs allows the user to conveniently perform single and bulk registrations of LWM2M devices. They also perform input validation and detect common errors before the device is created. See also [LWM2M > Registering LWM2M devices](https://cumulocity.com/guides/protocol-integration/lwm2m/##register-device) in the *Protocol integration guide*.
+
+The video snippets below demonstrate these new wizards.
+
+##### Single LWM2M device registration
+<video width="99%" controls>
+  <source src="/images/release-notes/lwm2m-reg.mp4" type="video/mp4" />
+</video>
+
+##### Bulk LWM2M device registration
+<video width="99%" controls>
+  <source src="/images/release-notes/lwm2m-bulk-reg.mp4" type="video/mp4" />
```

Comment:
@sag-eweingaertner is it possible to crop the video to remove the desktop items?
![Screenshot from 2022-08-15 09-55-55](https://user-images.githubusercontent.com/92035029/184590307-8f5696d2-a6c7-4c6d-915b-e2350eeb4030.png)